pages/WhatIsOppia.html

```diff
@@ -8,7 +8,7 @@
 
   <p>
     Oppia is an online learning tool that enables anyone to easily create and
-    share interactive activities ('explorations') which try to recreate the
+    share interactive activities (or <strong>explorations</strong>) which try to recreate the
```

Comment:
Reading this again, I think, maybe change 'or' to 'called'? Otherwise it reads as though explorations are an alternative to interactive activities, and that the two are mutually exclusive.